src/track/heaptrack_preload.cpp

```diff
@@ -175,7 +177,15 @@ extern "C" {
 
 /// TODO: memalign, pvalloc, ...?
 
-void* malloc(size_t size) noexcept
+// NOTE: adding noexcept to C functions is a hard error in clang++
```

Comment:
we should probably remove them altogethen then, but I can do that myself in a follow up